core/src/components/modal/gestures/sheet.ts

```diff
@@ -5,6 +5,32 @@ import { getBackdropValueForSheet } from '../utils';
 
 import { calculateSpringStep, handleCanDismiss } from './utils';
 
+export interface MoveSheetToBreakpointOptions {
+  /**
+   * The breakpoint value to move the sheet to.
+   */
+  breakpoint: number;
+  /**
+   * The offset value between the current breakpoint and the new breakpoint.
+   *
+   * For breakpoint changes as a result of a touch gesture, this value
+   * will be calculated internally.
+   *
+   * For breakpoint changes as a result of dynamically setting the value,
+   * this value should be the difference between the new and old breakpoint.
+   * For example:
+   * - breakpoints: [0, 0.25, 0.5, 0.75, 1]
+   * - Current breakpoint value is 1.
+   * - Setting the breakpoint to 0.25.
+   * - The offset value should be 0.75 (1 - 0.25).
+   */
+  breakpointOffset: number;
```

Comment:
This value is dynamic based on the execution context. When using `setCurrentBreakpoint`, we need to transition from the current breakpoint to the specified breakpoint. 

For the gesture behavior with the `canDismiss`, it needs to clamp to the nearest breakpoint. Since the value is different, it cannot be calculated internal to the `moveSheetToBreakpoint` function and needs to passed in/defined when calling the function.